clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/IngestionNotificationDispatcher.java

```diff
@@ -309,9 +309,12 @@ void reportError(Collection<PartitionConsumptionState> pcsList, String message,
         logMessage.append("Replica: ").append(pcs.getReplicaId()).append(" has already been ");
         boolean report = true;
 
+        // Report error for non-current version ingestion otherwise they will be stuck till push timeout
         if (pcs.isComplete()) {
-          logMessage.append("marked as completed so an error will not be reported.");
-          report = false;
+          if (pcs.isHybrid() || pcs.isCurrentVersion()) {
+            logMessage.append("marked as completed so an error will not be reported.");
```

Comment:
Should we only allow hybrid completed version to not report error? I am thinking there could be the case that this is the last record of the batch push partition and we had such error during reporting, then we don't have another record to check it again to mark it as ready-to-serve though, I think if that's true then we will still wait 24hrs.

Reply:
Actually I think a bit... it is not 24hr as it is not push job. But current version batch only store could either stuck in STARTED or ERROR. I think either case this would not block future job. 

How about we just don't trigger this error during batch push? I found that we don't have to if we don't even enter  reportIfCatchUpVersionTopicOffset(). Because for batch store, the following ready-to-serve will make sure it is ready and report completed directly.